src/cider/nrepl/middleware/stacktrace.clj

```diff
@@ -187,6 +188,37 @@
   (when-let [data (ex-data e)]
     (into {} (filter (comp (complement ex-data-blacklist) key) data))))
 
+
+(def spec-abbrev (delay
+                   (if (try (require 'clojure.spec.alpha) true
+                            (catch Throwable _ false))
+                     (resolve 'clojure.spec.alpha/abbrev)
+                     #'identity)))
+
+(defn prepare-spec-data [ed pprint-fn]
+  (let [pp-str #(with-out-str (pprint-fn %))
+        problems (sort-by #(count (:path %)) (:clojure.spec.alpha/problems ed))]
```

Comment:
I'm a bit worried about the hardcoded `alpha` namespaces.

Reply:
One way to fix this would be to create a correct namespace in try-catch from above and create an alias there. Will think of something.